advanced_functionality/tensorflow_bring_your_own/tensorflow_bring_your_own.ipynb

```diff
@@ -0,0 +1,743 @@
+    "\n",
+    "In addition, we must now specify the ECR image URL, which we just pushed above. Be sure to replace the string within the Estimator parameter, image_name.\n",
+    "\n",
+    "Finally, our local training dataset has to be in Amazon S3 and the S3 URL to our dataset is passed into the `fit()` call."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.estimator import Estimator\n",
+    "\n",
+    "hyperparameters = {'train-steps': 100}\n",
+    "\n",
+    "instance_type = 'ml.m4.xlarge'\n",
+    "\n",
+    "estimator = Estimator(role=role,\n",
+    "                      train_instance_count=1,\n",
+    "                      train_instance_type=instance_type,\n",
+    "                      image_name='ecr-image',\n",
+    "                      hyperparameters=hyperparameters)\n",
+    "\n",
+    "estimator.fit(data_location)\n",
+    "\n",
+    "predictor = estimator.deploy(1, instance_type)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'predictions': [{'probabilities': [0.115806,\n",
+       "    0.119459,\n",
+       "    0.028497,\n",
+       "    0.348986,\n",
+       "    0.102692,\n",
+       "    0.0354596,\n",
+       "    0.0917221,\n",
+       "    0.00540253,\n",
+       "    0.121872,\n",
+       "    0.0301034],\n",
+       "   'classes': 3}]}"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "image = cv2.imread(\"data/cat.png\", 1)\n",
+    "\n",
+    "# resize, as our model is expecting images in 32x32.\n",
+    "image = cv2.resize(image, (32, 32))\n",
+    "\n",
+    "data = {'instances': numpy.asarray(image).astype(float).tolist()}\n",
+    "\n",
+    "predictor.accept = 'application/json'\n",
+    "predictor.content_type = 'application/json'\n",
+    "\n",
+    "predictor.serializer = json_serializer\n",
+    "predictor.deserializer = json_deserializer\n",
+    "\n",
+    "predictor.predict(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Making predictions using Boto3\n",
+    "\n",
+    "Below is an example of making an inference request through Boto3 on the same endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "b'{\\n    \"predictions\": [\\n        {\\n            \"classes\": 3,\\n            \"probabilities\": [0.122724, 0.0958609, 0.0519071, 0.272535, 0.097384, 0.0535893, 0.0905842, 0.0250508, 0.123435, 0.0669298]\\n        }\\n    ]\\n}'\n"
```

Comment:
I think we should add some markdown describing how to interpret the predictions.